src/bitcoin-client.js

```diff
@@ -85,6 +102,54 @@ async function createAndBroadcastTx(keyPair, address, value, changeAddress) {
   return txId;
 }
 
+/**
+ * Creates and broadcasts a tx to send bitcoin to the recipient address.
+ *
+ * The broadcast may fail if i.e. an UTXO has too many unconfirmed ancestors.
+ * For that reason and only in that case (the node returns the error code -26),
+ * the operation is retried with different selection strategies.
+ *
+ * Selecting the smaller value UTXOs first helps reducing the risk of creating
+ * dust but the cost may be higher until all the those small transactions are
+ * spent.
+ *
+ * Selecting UTXOs at random reduces the risk of selecting the same UTXOs in
+ * multiple concurrent operations, preventing the creation of long UTXO chains
+ * that would cause the node to reject the tx.
```

Comment:
minor spelling improvements 😄 

```suggestion
 * Creates and broadcasts a tx to send Bitcoin to the recipient address.
 *
 * The broadcast may fail if i.e. an UTXO has too many unconfirmed ancestors.
 * For that reason and only in that case (the node returns the error code -26),
 * the operation is retried with different selection strategies.
 *
 * Selecting the smaller value UTXOs first helps to reduce the risk of creating
 * dust but the cost may be higher until all those small transactions are
 * spent.
 *
 * Selecting UTXOs at random reduces the risk of selecting the same UTXOs in
 * multiple concurrent operations, preventing the creation of long UTXO chains
 * that would cause the node to reject the tx.
```

Reply:
"Bitcoin" is the network while "bitcoin" is the money. Thanks pointing out the other typos!